File: Q06/main.c
#include<stdio.h>
int main(){
    int n;
    scanf("%d",&n);
    int arr[n];
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }

    int k;
    scanf("%d",&k);

    int start=0;

    for(int i=0;i<n;i++){
        if(arr[i]!=k){
            arr[start]=arr[i];
            start++;
        }
    }
    
    for(int i=0;i<start;i++){
        printf("%d ",arr[i]);
    }
    return 0;
}

File: Q06/main.py
n=int(input())
arr=[]
for i in range(n):
    arr.append(int(input())) 

k=int(input())
start=0

for i in range(n):
    if arr[i]!=k:
        arr[start]=arr[i]
        start+=1

for i in range(start):
    print(arr[i],end=" ")